.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,31 +1,2 @@
-<!--- 👉👉👉 Provide a general summary of your changes. 👈👈👈 -->
-
-## Motivation and Context
-<!--- 💭💡💭 Why is this change required? What problem does it solve? 💭💡💭 -->
-<!--- 🔗 If it fixes an open issue, please link to the issue here. 🔗 -->
-
-## Description
-<!--- 📝🎯📝 Describe your changes in detail. 📝🎯📝 -->
-
-## How has this been tested?
-<!--- 🙆👌🙆 Please describe in detail how you tested your changes. 🙆👌🙆 -->
-<!--- Include details of your testing environment, details of your manual tests, snippet code or commands of your manual tests, table of experiments results metrics, tests ran to see how your change affects other areas of the code, etc. -->
-<!--- If you have experiments report documents please link to here. -->
-
-## Screenshots (if appropriate):
-
-## Types of changes
-<!--- 🏁🎌🏁 What types of changes does your code introduce? Put an `x` in all the boxes that apply: 🏁🎌🏁 -->
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature / Optimization (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-
-## Checklist:
-<!--- ✅✅✅ Go over all the following points, and put an `x` in all the boxes that apply. ✅✅✅ -->
-<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
-
-<!--- TODO(wakisaka): After decide our code style, add this.
-- [ ] My code follows the code style of this project.
--->
-- [ ] My change requires a change to the documentation.
-- [ ] I have updated the documentation accordingly.
+Dcribe what this patch does to fix the issue.
```

Comment:
Thanks! Please fix typo. And I think we'd better highlight this line as below.
## Describe what this patch does to fix the issue.